lib/convert/string/convertSrtToVtt.ts

```diff
@@ -0,0 +1,32 @@
+/**
+ * Convert SRT transcript to WebVTT transcript.
+ * @param srt SRT transcript string.
+ * @returns WebVTT transcript string.
+ */
+const convertSrtToVtt = (srt: string) => {
```

Comment:
Do these then get converted to JSON, which is what the browser is using?  (So we're always going `srt -> vtt -> json`)

Reply:
The VTT endpoint stops at VTT. Browser track elements will use the VTT data. JSON endpoint will go to JSON. JSON data is used to provide the high-fidelity highlighting and are loaded separately only when CC view is opened or episode is viewed on listen page. The index endpoint goes to a custom data model to facilitate rendering requirements of the transcript on episode pages.